src/Microsoft.PowerShell.Commands.Utility/commands/utility/Tee-Object.cs

```diff
@@ -72,6 +73,16 @@ public SwitchParameter Append
 
         private bool _append;
 
+        /// <summary>
+        /// Gets or sets the Encoding.
+        /// </summary>
+        [Parameter(Mandatory = false, ParameterSetName = "File")]
+        [Parameter(Mandatory = false, ParameterSetName = "LiteralFile")]
```

Comment:
Please remove default values:
```suggestion
        [Parameter(ParameterSetName = "File")]
        [Parameter(ParameterSetName = "LiteralFile")]
```